src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -111,8 +111,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('hub_name', hub_name_type, options_list=['--name', '-n'], id_part='name')
         c.argument('etag', options_list=['--etag', '-e'], help='Entity Tag (etag) of the object.')
         c.argument('sku', arg_type=get_enum_type(IotHubSku),
-                   help='Pricing tier for Azure IoT Hub. Default value is F1, which is free. '
-                        'Note that only one free IoT hub instance is allowed in each '
+                   help='Pricing tier for Azure IoT Hub. '
+                        'Note that only one free IoT hub instance (F1) is allowed in each '
```

Comment:
It's no longer necessary to declare that the default value is S1?

Reply:
It's not required to mention default value as cli-core extract the default value from arg parsing.
The help content for SKU will look like: 
![image](https://user-images.githubusercontent.com/31940305/76583090-7d1a4600-6495-11ea-895a-92908817dae4.png)